app/lib/common/utils/utils.dart

```diff
@@ -369,15 +369,12 @@ Color getUserAvatarNameColor(types.User user, List<Color> colors) {
 
 String getUserInitials(types.User user) {
   var initials = '';
-
-  if ((user.firstName ?? '').isNotEmpty) {
-    initials += user.firstName![0].toUpperCase();
-  }
-
-  if ((user.lastName ?? '').isNotEmpty) {
-    initials += user.lastName![0].toUpperCase();
-  }
-
+  user.firstName.let((p0) {
+    if (p0.isNotEmpty) initials += p0[0].toUpperCase();
+  });
+  user.lastName.let((p0) {
+    if (p0.isNotEmpty) initials += p0[0].toLowerCase();
+  });
```

Comment:
I don't feel like using `p0` is a very nice way of describing these. Let's stick to using proper words that make it easier to follow along later:
```suggestion
  user.firstName.let((name) {
    if (name.isNotEmpty) initials += name[0].toUpperCase();
  });
  user.lastName.let((name) {
    if (name.isNotEmpty) initials += name[0].toLowerCase();
  });
```

Reply:
Yeah, I just wanted to use the word that has universal meaning, so that other developers can copy/paste and use this code instantly.

Reply:
@gnunicorn Do you want to use specialized word not universal word?

Reply:
I prefer explicit words, yes.

Reply:
Frankly, I wanted to use `p0` for object and `cb` for callback.
Don't worry.
Will fix.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2233/commits/68d5e20f88fd02b863a771d7b583630301a41158

Reply:
cb for callback is specific enough.